src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -173,8 +176,11 @@ def create_request(
         # Get the query from 'create_query'
         es_query = ElasticTransformDump.create_query(filters)
         # Do a post_filter using the returned query
-        es_search = es_search.query(
-            es_query) if not post_filter else es_search.post_filter(es_query)
+        es_search = es_search.query(es_query) if not post_filter else es_search.post_filter(es_query)
```

Comment:
Can you convert this to a if-else statement? I feel that would make this important conditional more prominent.